core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -214,7 +226,19 @@ var TopicsAndSkillsDashboardPage = function() {
       confirmSkillCreationButton,
       'Create skill button takes too long to be clickable');
     confirmSkillCreationButton.click();
-    waitFor.pageToFullyLoad();
+
+    waitFor.newTabToBeCreated('Creating skill takes too long');
+    return browser.getAllWindowHandles().then(function(handles) {
+      var firstTabHandle = handles[0];
+      var secondTabHandle = handles[1];
+      return browser.switchTo().window(secondTabHandle).then(function() {
+        waitFor.pageToFullyLoad();
```

Comment:
ditto: why waiting here instead of for tab we switch to at the end

Reply:
Done